lib/cisco_node_utils/vxlan_global.rb

```diff
@@ -25,29 +26,13 @@ module Cisco
   class VxlanGlobal < NodeUtil
     # Constructor for vxlan_global
     def initialize(instantiate=true)
-      enable if instantiate && !VxlanGlobal.enabled
-    end
-
-    def enable
-      config_set('vxlan_global', 'feature', state: '')
+      Feature.fabric_forwarding_enable if instantiate
     end
 
     def disable
-      config_set('vxlan_global', 'feature', state: 'no')
       dup_host_mac_detection_default
     end
 
-    # Check current state of the configuration
-    def self.enabled
-      feat = config_get('vxlan_global', 'feature')
-      return !(feat.nil? || feat.empty?)
-    rescue Cisco::CliError => e
-      # This cmd will syntax reject if feature is not
-      # enabled. Just catch the reject and return false.
-      return false if e.clierror =~ /Syntax error/
-      raise
```

Comment:
I think it was copied over from bgp, which originally did a 'show run bgp', which fails when the feature is off. If the yaml just checks 'show running | section feature' then it won't fail.
So yeah just delete it.